services/people.rb

```diff
@@ -0,0 +1,41 @@
+require 'lib/ir_pagerduty'
+
+class People
+  def fetch(path, params={})
+    pd = IRPagerduty.new
+    pd.fetch_json(path, params)
+  end
+
+  def fetch_users(params)
+    fetch("users", params)['users']
+  end
+
+  def fetch_schedules(params)
+    fetch("schedules", params)['schedules']
+  end
+
+  def fetch_schedules_users(schedule_id, params)
+    fetch("schedules/#{schedule_id}/users", params)['users']
+  end
+
+  def fetch_users_contact_methods(user_id)
+    fetch("users/#{user_id}/contact_methods")['contact_methods']
+  end
+
+  def fetch_irms
+    today = Date.today
+    users = fetch_schedules_users(SupportApp.pager_duty_irm_schedule_id, {
```

Comment:
Rather than accessing SupportApp.pager_duty_irm_schedule_id directly could you pass it in to the constructor and then use `@schedule_id` here?